dataService.js

```diff
@@ -0,0 +1,91 @@
+// Maps
+let productMap;
+let categoryMap;
+let combinedProductMap;
+
+/*
+ * Method to create and return map of products if its not already created
+ */
+function getProducts() {
+
+  if(productMap) {
```

Comment:
Run eslint again here. 
```suggestion
  if (productMap) {
```